web3.js/src/publickey.ts

```diff
@@ -150,6 +150,13 @@ export class PublicKey {
     }
     throw new Error(`Unable to find a viable program address nonce`);
   }
+
+  /**
+   * Check that a pubkey is on the ed25519 curve.
+   */
+  static isOnCurve(pubkey: any): boolean {
```

Comment:
I still think this needs a type, should be a byte array right? We can probably use `Unit8Array`

Reply:
Should that be something like `Uint8Array | Buffer`?  What's the right way to get a construct pubkey into one of these?

Reply:
`Uint8Array` is good!